src/OtpInput/OtpInput.types.ts

```diff
@@ -10,7 +10,7 @@ export interface OtpInputProps {
   focusStickBlinkingDuration?: number;
   secureTextEntry?: boolean;
   theme?: Theme;
-}
+} & TextInputProps;
```

Comment:
I would prefer to dedicate `TextInputProps` into a separate prop like (textInputProps: TextInputProps) rather than populating the whole component props :)